test/api/artifact_test.js

```diff
@@ -315,6 +315,66 @@ suite('Post artifacts', function() {
     });
   });
 
+  test("Check expire doesn't drop table", async () => {
```

Comment:
Extra that snuck in.... Feel free to ignore... Basically, I realized when writing expiration tests for taskgroup thing, that our tests for expiration only tests that things gets deleted... They don't test that things that aren't suppose to expire doesn't get deleted...

If by accident we flipped  `expires <= now()` into `expires >= now()` then we would effectively drop the entire table :)
I think that would be bad... So these tests are just to guard against that...